neural_compressor/adaptor/pytorch.py

```diff
@@ -917,6 +917,7 @@ def model_calibration(self, q_model, dataloader, iterations=1, conf=None, calib_
                     dataloader.batch(1)
                     self.calib_func(q_model, dataloader, calib_sampling_size, conf)
             else:  # pragma: no cover
+                dataloader.batch_size = getattr(dataloader, "batch_size") or getattr(dataloader, "total_batch_size")
```

Comment:
we can only fetch batch_size info but not set it to dataloader

Reply:
fixed in [d625e0a](https://github.com/intel/neural-compressor/pull/1219/commits/d625e0a69371ee82aad480f71fb9e6ac75a5ea16)